android/src/main/java/nl/xguard/flic2/service/Flic2Service.java

```diff
@@ -55,11 +74,15 @@ public void onCreate() {
             notificationManager.createNotificationChannel(mChannel);
         }
 
-        Notification notification = new Builder(getApplicationContext(), NOTIFICATION_CHANNEL_ID)
+        notification = new Builder(getApplicationContext(), NOTIFICATION_CHANNEL_ID)
```

Comment:
When you don't add the meta data it sets no text at all, it should have the basic text like before. When you dont set icon is takes the icon from your app it should also do this when you dont set it specifically.